src/Oro/Bundle/DataGridBundle/Resources/doc/backend/extensions/formatter.md

```diff
@@ -12,6 +12,7 @@ column_name:
     type: field # default value `field`, so this key could be skipped here
     frontend_type: date|datetime|decimal|integer|percent|currency|select|text|html|boolean # optional default string
     data_name: someAlias.someField # optional, key in result that should represent this field
+    divisor: some number # optional if you need to divide a numeric value by a number before rendering it
```

Comment:
The value `100` will make sense for money if you want to store the amount as Cent (integer) into database. I think this is the most common use case. Then for instance there is `1042` persisted in database, and datagrid will show `10.42`, the value in Dollars.